sched/pthread/pthread_condwait.c

```diff
@@ -88,7 +89,7 @@ int pthread_cond_wait(FAR pthread_cond_t *cond, FAR pthread_mutex_t *mutex)
 
       sinfo("Give up mutex / take cond\n");
 
-      cond->wait_count++;
+      atomic_fetch_add((FAR atomic_t *)&cond->wait_count, 1);
```

Comment:
let's add COND_WAIT_COUNT macro in sched/pthread/pthread.h like NXSEM_COUNT to simplify the usage?